src/main/webapp/script.js

```diff
@@ -171,139 +246,139 @@ function getLoginStatus() {
 
 }
 
+// test function for displaying recipes
+function hardCodedRecipeCard() {
+  const displayRecipeElement = document.getElementById('display-recipes');
+  displayRecipeElement.innerHTML = "";
+  
+  const recipe = {}
+  recipe['id'] = 1;
+  recipe['title'] = "Title";
+  recipe['image'] = "/images/salad.jpeg";
+  recipe['sourceUrl'] = "https://css-tricks.com/snippets/css/a-guide-to-flexbox/";
+  recipe['vegetarian'] = true;
+  displayRecipeElement.appendChild(createRecipeElement(recipe));
+}
+
 /** Creates an element that represents a recipe card */
-// TODO: will change to use html template element
 function createRecipeElement(recipe) {
-  const recipeElement = document.createElement('div');
-  recipeElement.className = 'recipe-card';
-
-  const titleElement = document.createElement('p');
+  var temp = document.querySelector("#recipe-template");;
+  var clone = temp.content.cloneNode(true);
```

Comment:
maybe change the name to something more descriptive, showing what it is a clone of